File: Cell/cellcs/Eucariota.cs
public class Eucariota : Celula{
    private Boolean nucleo = true;

    public Eucariota(Boolean nucleo){
        this.nucleo = nucleo;
    }
    public Eucariota(){}

    public Boolean getNucleo() {
        return nucleo;
    }

    public void setNucleo(Boolean nucleo) {
        this.nucleo = nucleo;
    }

    public new String toString(){
        return base.toString() + "Existe nucleo? "+nucleo+"\n";
    }    
}

File: Cell/Eucariota.java
public class Eucariota extends Celula{
    private boolean nucleo = true;

    public Eucariota(boolean nucleo) {
        this.nucleo = nucleo;
    }
    
    public Eucariota(){}
    
    public boolean getNucleo() {
        return nucleo;
    }

    public void setNucleo(boolean nucleo) {
        this.nucleo = nucleo;
    }

    public String toString(){
        return super.toString() + "Existe nucleo? "+nucleo+"\n";
    }    
}
